src/promptflow/tests/sdk_cli_azure_test/e2etests/test_azure_cli_perf.py

```diff
@@ -6,18 +7,40 @@
 import pytest
 
 from promptflow._cli._user_agent import USER_AGENT as CLI_USER_AGENT  # noqa: E402
+from promptflow._sdk._telemetry import log_activity
 from promptflow._sdk._utils import ClientUserAgentUtil
+from sdk_cli_azure_test.recording_utilities import is_replay
 
 FLOWS_DIR = "./tests/test_configs/flows"
 DATAS_DIR = "./tests/test_configs/datas"
 
 
+def mock_log_activity(*args, **kwargs):
+    custom_message = "flow run: https://github.com/microsoft/promptflow/actions/runs/{0}".format(
+        os.environ.get("FLOW_RUN_ID")
+    )
+    if len(args) == 4:
+        if args[3] is not None:
```

Comment:
Add a screenshot on query result after this change?

Reply:
Ok, I will do it.
But I have a question, is our telemetry data considered private? Can you take a screenshot and place it in a public place?

Reply:
![image](https://github.com/microsoft/promptflow/assets/23182548/22898b27-dc7f-4178-a9a9-55d51fac3d0e)
https://github.com/microsoft/promptflow/actions/runs/7334689811

Reply:
![image](https://github.com/microsoft/promptflow/assets/23182548/51af569f-aa33-49dc-9d0a-aa348aadb795)